### DISCUSSION TITLE
Command timeouts as there is no output

### DISCUSSION THREAD

**Discussion by 5u623l20** [cirruslabs/cirrus-ci-docs#1239] [Q&A]
Hi,
I am doing a CI test running the [Vale](https://vale.sh) command which do not generate any outputs until it finishes. For our corner case it takes around 15-20 minutes before generating an output. Due to this the task is timing out. Is there any way to prevent it?

**Comment by fkorotkov** [comment]
Could you please provide a link to a failing task? No output should not abort your tasks,

**Comment by 5u623l20** [comment]
https://cirrus-ci.com/task/5852048961306624

**Reply by fkorotkov** [reply]
It does seems like CPU is active and your task is doing something. Have you considered to increase the [timeout from `20m`](https://github.com/5u623l20/freebsd-doc/blob/97be5345f606de1d7e10297a977fc4c1b0dcf7b0/.cirrus.yml#L8)?

**Reply by 5u623l20** [reply]
I didn't try the timeout as I thought that it was by default 60m or has anything changed? And it was at 20m when this timed out.

**Reply by 5u623l20** [reply]
Arrgh damn. Sorry for the noise.
